client/cgo/cgotest.go

```diff
@@ -0,0 +1,235 @@
+package main
+
+/*
+int testVRFVerify(unsigned char *pk, int pkSize,
+    unsigned char *m, int mSize,
+    unsigned char *index, int indexSize,
+    unsigned char *proof, int proofSize) {
+    if (cgoVrfVerify(pk, pkSize, m, mSize,
```

Comment:
I just re-read my code and realized that I missed something. What I mean when using these C functions as bridges is that I want to make the same scenario as when the lib being used in the add-on with js-ctypes. That is the input should be type `unsigned char*` and passed to go functions that take arguments type `unsafe.Pointer`. Hopefully it makes sense to you.